src/components/singleCheck.tsx

```diff
@@ -0,0 +1,71 @@
+interface CheckSelectProps {
+  selectTitle: string;
+  selectDesc: string;
+  isChecked: boolean;
+  onSelectionChange: (isChecked: boolean) => void;
+  isDisabled?: boolean;
+}
+
+const SingleCheckSelect: React.FC<CheckSelectProps> = ({
+  selectTitle,
+  selectDesc,
+  isChecked,
+  onSelectionChange,
```

Comment:
I think would be better to set `isDisabled` with the default value set to `false`. However, I just realized that there doesn't seem to be a disabled logic implemented in the code, making the `isDisabled` prop unused, so can try to add that